### ISSUE TITLE
Thief Detection : Person's Behavior / Actions

### ISSUE THREAD

**Issue by prathimacode-hub** [neha07kumari/Thief_Detection#9]
### Description

Welcome to 'Thief -Detection', good to see you here.

With this issue, the readers will get to know on how to detect a thief with person's behavior and actions. Add 1-2 inputs for it.

It's an interlinked issue with earlier issue of Thief Detection. Discuss with the collaborator assigned to issue number - 6 and work accordingly

To get assigned to this issue, add your participant id mentioned in this [spreadsheet](https://docs.google.com/spreadsheets/d/1SX9FubA-n1ag_0n4WmzMXd6rq5lrkFmaAWl8fhHvTWM/edit#gid=0) and the approach you would follow.

Points to Note :

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.

- "Issue Title" and "PR Title" should be the same. Include issue number in PR template along with it.

- You should add up in respective folder according to the demands of the issue.

- Inclusion of Screenshot is must to approve 

This issue is only for 'SWOC' contributors

### Domain

Data Science

### Code of Conduct

- [X] I  will follow [Contributing Guidelines](https://github.com/neha07kumari/Thief_Detection/blob/main/CONTRIBUTING.md) & [Code of conduct](https://github.com/neha07kumari/Thief_Detection/blob/main/CODE_OF_CONDUCT.md) of this project.

**Comment by kkg2001** [comment]
I want to work on this issue #9

**Comment by Vanshikagoel0012** [comment]
Hey i want to take this pr.

**Comment by prathimacode-hub** [comment]
This is an issue not a PR. Ok, so tell me your approach and participant id to get it assigned. @Vanshikagoel0012

**Comment by Vanshikagoel0012** [comment]
okay, I will use unsupervised learning 
participant id-1093

**Comment by prathimacode-hub** [comment]
Ok, issue assigned to @Vanshikagoel0012. As mentioned, discuss with collaborator assigned to issue number - 6 and work accordingly since issue is interlinked. Add 1-2 inputs which are person's behavior and actions to get this done. @Vanshikagoel0012

**Comment by Vanshikagoel0012** [comment]
oka

**Comment by prathimacode-hub** [comment]
Its been nearly more than 20days. Kindly update the progress and it should be done by now. If you're stuck anywhere, we're here to help out. @Vanshikagoel0012

**Comment by Vanshikagoel0012** [comment]
oka will submit it by tommorrow

**Comment by Vanshikagoel0012** [comment]
Hey @prathimacode-hub can you please give me layout how to get started on this as this don't have any dataset

**Comment by Vanshikagoel0012** [comment]
??? @prathimacode-hub

**Comment by prathimacode-hub** [comment]
Yes, this issue is continuation of person who is detected as thief or not. Based on intruder's action, we detect him to be a thief. Especially a thief uses a short cut methods or use unsual path, get tensed. Here we can also include computer vision too to see facial recognitions @Vanshikagoel0012

**Comment by prathimacode-hub** [comment]
So you focus on behavior and actions as unusual path is already done. You can get any public datasets which has facial recognitions that are relevant to this context. @Vanshikagoel0012

**Comment by Vanshikagoel0012** [comment]
oka,thank you!

**Comment by Vanshikagoel0012** [comment]
I just have to upload a python file and dataset right?

**Comment by prathimacode-hub** [comment]
Yes, actually you have to integrate in the existing code and dataset is uploaded separately in respective space. @Vanshikagoel0012

**Comment by Vanshikagoel0012** [comment]
I am not able to find dataset for this and where I have to integrate I didn't get it please explain

**Comment by prathimacode-hub** [comment]
You can get it in Kaggle. There are facial recognition datasets. Download them and filter it according to behavioral actions that might help us in this project. Integrating means into the main file. Its all a single project so it should be done in the existing file. Create your application separately and embed it into main file. You can refer to the existing code. @Vanshikagoel0012

**Comment by Vanshikagoel0012** [comment]
So basically I have to get a dataset from anywhere and then train it for facial recognition and then add the python file only to the main file right?

**Comment by prathimacode-hub** [comment]
Yes exactly. @Vanshikagoel0012

**Comment by Vanshikagoel0012** [comment]
hey, I have submitted my pull request please have a look and tell me if any changes you want @prathimacode-hub
